config/beginner.yaml

```diff
@@ -0,0 +1,70 @@
+hash_length: 7
+container_framework: docker
```

Comment:
This is a reminder for all of us that we should freeze development for the 1-2 week period before the tutorial. For instance, #387 changes this syntax, and we don't want to have the tutorial files be slightly out of date. That would be quite confusing.